controller/scheduler/main.go

```diff
@@ -746,10 +757,19 @@ func (f *Formation) jobType(job *host.Job) string {
 // sortJobs sorts Jobs in reverse chronological order based on their startedAt time
 type sortJobs []*Job
 
-func (s sortJobs) Len() int           { return len(s) }
-func (s sortJobs) Less(i, j int) bool { return s[i].startedAt.Sub(s[j].startedAt) > 0 }
-func (s sortJobs) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
-func (s sortJobs) Sort()              { sort.Sort(s) }
+func (s sortJobs) Len() int { return len(s) }
+func (s sortJobs) Less(i, j int) bool {
+	switch {
+	case s[i].timer != nil && s[j].timer == nil:
+		return true
+	case s[i].timer == nil && s[j].timer != nil:
```

Comment:
`sortJobs` is used then removing jobs due to scaling, whereas `Job.timer` is potentially written to when the host job crashes (which will be in another goroutine), so I think it does need a lock here.